cli/client.go

```diff
@@ -765,7 +765,7 @@ func newViamClient(c *cli.Context) (*viamClient, error) {
 	conf, err := ConfigFromCache()
 	if err != nil {
 		if !os.IsNotExist(err) {
-			return nil, err
+			return nil, errors.New("Failed to parse cached config. Please log in again.")
```

Comment:
Lint's probably gonna complain about uppercased error. We automatically uppercase errors from the CLI in output, anyway.